fleet-management-persistence/src/main/java/cz/fi/muni/pa165/entity/VehicleCategory.java

```diff
@@ -15,7 +15,6 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    @NotNull
```

Comment:
nie ale je to zadefinovane v @Column(nullable = false) hned riadok pod, @NotNull robilo to ze to skontroloval hibernate este pred vlozenim do databazy a vyhodil vynimku taku ze ju postProcessor nevedel prelozit na DataAccessException a neprechadzal test,